plugin/storage/kafka/options.go

```diff
@@ -141,6 +148,21 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		defaultCompressionLevel,
 		"(experimental) compression level to use on messages. gzip = 1-9 (default = 6), snappy = none, lz4 = 1-17 (default = 9), zstd = -131072 - 22 (default = 3)",
 	)
+	flagSet.Duration(
+		configPrefix+suffixBatchLinger,
+		defaultBatchLinger,
+		"(experimental) Number of milliseconds to delay before sending records to Kafka. Higher value reduce request to Kafka but increase latency and the possibility of data loss in case of process restart. See https://kafka.apache.org/documentation/",
```

Comment:
s/Number of milliseconds to delay/Time interval to wait/

Reply:
Changed